scripts/jenkins-acceptance-tests.sh

```diff
@@ -37,3 +37,10 @@ pip install -r requirements/test-acceptance.txt
 
 cd test/acceptance
 python tests.py
+
+# Unset SELENIUM_HOST so that bok-choy doesn't try to use saucelabs
+unset SELENIUM_HOST
+# AutoAuthPage times out in PhantomJS when using https, switch to use http
```

Comment:
Interesting...I'm curious how old your devstack is... There were some TLS/SSL changes to buildouts...maybe 6 months ago?